packages/v4/src/content/design-guidelines/components/drag-and-drop/drag.md

```diff
@@ -0,0 +1,72 @@
+---
+id: Drag and drop
+section: components
+---
+The **drag and drop** interaction can be used to reposition elements on screen into a layout that benefits the user. This gives the user more flexibility to arrange and/or group items without having to make code changes.
+
+## Usage
+### Simple list
+1. Draggable area: The drag and drop interaction can be triggered with click and hold over any part of the item. The fa-grip icon is used to show that the drag and drop interaction is available.
```

Comment:
Could you bold every first few words before the semi colon when you have numbered lists?

So` 1. **Draggable area**: the drag....`

Reply:
Also might be helpful to use the "code" styling for "fa-grip", so '&fa-grip'